tests/patch.go

```diff
@@ -0,0 +1,39 @@
+// From https://gist.github.com/imosquera/6716490#sthash.O4z2aQQp.LUHz2Cbb.dpuf
```

Comment:
heketi/tests is a separate package from heketi/heketi and it doesn't import anything outside the Go stdlib. It is okay to import it here, instead of copying.